docs/project_droneworld/about.md

```diff
@@ -1,58 +1,54 @@
+
+By automating and streamlining the testing process, Drone World improves safety, reliability, and efficiency for drone developers. It allows comprehensive pre-flight testing in ultra-realistic environments, helping developers refine their systems and iterate more rapidly on complex missions. Our team at OSS is dedicated to enhancing Drone World’s capabilities, including refining environmental settings, drone configurations, and integrating new features.
 
 ### Information
 
 - **Source Code:**  https://github.com/oss-slu/DroneWorld [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/DroneWorld)
-- **Client** [Dr.Ankit Agrawal](https://www.slu.edu/science-and-engineering/academics/computer-science/faculty-and-staff/ankit-agrawal.php) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ankyAgrawal)
-- **Current Tech Lead:** [Yashaswini Shivalingaiah](https://yashaswinishivalingaiah.com/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashaswini-slu)  [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashaswini-shivalingaiah-467a9652/)
+- **Client:** [Dr. Ankit Agrawal](https://www.slu.edu/science-and-engineering/academics/computer-science/faculty-and-staff/ankit-agrawal.php) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ankyAgrawal)
+- **Current Tech Lead:** Mohammad Al-Hanoosh [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/mohamdlog)  <!--[<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/)-->
 - **Developers:**
-    - Kaleb Yu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/kalyus)
-    - Szymon Rostkowski [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/sr259)
-    - Hebron Bekele [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/hebronh) 
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/)
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/)
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/) 
 - **Start Date:** 9th Feb 2023
 - **Adoption Date:** 25 Aug 2023
-- **Technologies Used:**
-    - Tech Stack: Simulation, Testing, Visualization, Autonomous Systems
-    - Key Languages: Python, TypeScript with JavaScript
-    - Key Frameworks: React (Frontend), Flask (Backend APIs), Unreal Engine (Simulation)
-    - Databases: MongoDB 
-    - Critical Dependencies: 
-        - PX4 and Ardupilot
-        - Unreal Engine 
+
+### Technologies and Dependencies
```

Comment:
I'm not necessarily opposed to moving this to a separate section, but I want to maintain stylistic and structural consistency across portfolio pages. Please either revert this for now, or get all the other Tech Leads to make similar changes in their pages.